lib/developer_portal/app/assets/stylesheets/stats.scss

```diff
@@ -1,3 +1,4 @@
+@import 'stats/c3';
```

Comment:
Is this file uploaded to the CMS or loaded from our assets?

Reply:
That one is loaded from our assets stored in cloudfront... However, I've checked the minified version currently stored and it includes it... will keep investigating